src/components/searchResults/searchResults.jsx

```diff
@@ -3,65 +3,52 @@ import { hot } from "react-hot-loader";
 import "./searchResults.scss";
 import MovieList from "../movieList/movieList";
 import NoResults from "../noResults/noResults";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
 
 
 class SearchResults extends React.PureComponent {
-    constructor (props) {
-        super(props);
-        this.state = {}; // is necessary to show result/no result part
-    }
-
     render () {
+        const {
+            moviesList,
+            selectedSortOption
+        } = this.props;
+
+
+        const formattedMoviesList = moviesList.map((movie) => {
+            movie.release = new Date(movie.release_date).getUTCFullYear();
+            return movie;
+        });
+
+        moviesList.sort((option1, option2) => option2[selectedSortOption] - option1[selectedSortOption]);
```

Comment:
props are read-only, do not modify them, please

Reply:
done